### DISCUSSION TITLE
About V2 Wallet

### DISCUSSION THREAD

**Discussion by gityms123** [WalletConnect/walletconnect-monorepo#1226] [Developer Support]
Hello everyone, I want to make a V2 Android wallet now, but now I am testing the WalletConnectKotlinV2 source code and found that I can't even pass the most basic stellarx login QR code. The parameters in the QR code and the parameters in the code are mostly wrong. on, how should I handle it?

**Comment by hackyguru** [comment]
Hey, can I know the version of your Kotlin SDK? This might be because you are using an unstable version. FYI, v2 is still in beta.

**Reply by gityms123** [reply]
> Hey, can I know the version of your Kotlin SDK? This might be because you are using an unstable version. FYI, v2 is still in beta.

Hi, I have been testing the latest source code. The version should be 1.0.0-beta102 now. The parameters of the QR code of the 1.0.0-beta04 version a long time ago are still the same.
